.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -45,77 +45,3 @@ body:
       description: |
         A clear and concise description of any alternatives you've considered or any workarounds that are currently in
         place.
-  - type: textarea
-    attributes:
-      label: "System information and logs"
-      description: |
-        Provide details about the environment you experienced the issue in.
-      value: |
-        ### Environment
-        * broker version: please run `snap info authd-msentraid`
-        * authd version: please run `/usr/libexec/authd version`
-        * gnome shell version: please run `apt policy gnome-shell`
-        * Distribution: (**NAME** in `/etc/os-release`)
-        * Distribution version: (**VERSION_ID** on `/etc/os-release`):
-
-        ### Log files
-        Please redact/remove sensitive information:
-
-        #### Authd entries:
-
-        ```raw
-        journalctl -u authd.service
-        ```
-
-        ####  MS Entra ID broker entries:
-
-        ```raw
-        journalctl -u snap.authd-msentraid.authd-msentraid.service
-        ```
-
-        ####  Google broker entries:
-
-        ```raw
-        journalctl -u snap.authd-google.authd-google.service
-        ```
-
-        ### Application settings
-
-        Please redact/remove sensitive information:
-
-        ####  MS Entra ID broker configuration:
-
-        ```raw
-        cat /var/snap/authd-msentraid/current/broker.conf
-        ```
-
-        ####  MS Entra ID broker authd configuration:
-
-        ```raw
-        cat /etc/authd/brokers.d/msentraid.conf
-        ```
-
-        ####  Google broker configuration:
-
-        ```raw
-        cat /var/snap/authd-google/current/broker.conf
-        ```
-
-        ####  Google broker authd configuration:
-
-        ```raw
-        cat /etc/authd/brokers.d/google.conf
-        ```
-
-  - type: textarea
-    attributes:
-      label: Relevant information
-      description: >
-        Please look at our [Troubleshooting guide](https://github.com/ubuntu/authd/wiki/05--Troubleshooting) and provide logs for both authd and the oidc provider package.
-      placeholder: Remember to redact any sensitive information from them.
```

Comment:
Maybe we should keep the `Relevant Information` field but apply some rewording to it? I think it might be useful for users to (hopefully) provide some additional context on why they think the feature is a good thing or necessary.

Reply:
Mmh, do the remaining fields `Describe the feature`, `Describe the ideal solution`, and `Alternatives and current workarounds` not provide enough opportunity for describing the context of the feature? If not, what kind of context do you expect that doesn't fit in the other fields?

Reply:
I'd expect the users to provide some optional content here, like URLs for some specific documentation, other projects where the feature is already applied/being worked on, etc.

Reply:
The problem I have with the many forms in the template is that if they are left empty, they result in sections with *No response*, which I find distracting, especially if there are so many of them (see for example https://github.com/ubuntu/authd/issues/886). 

Even more annoying: When I create a new issue, until recently I was able to skip the template by using https://github.com/ubuntu/authd/issues/new?assignees=&labels=feature&projects=, but now that just redirects to the page where I have to choose a template. So now when I create a new issue, I have to manually remove the empty sections after creating the ticket.

Reply:
Fair enough. Let's remove it then and see how the impacts go